vendor/visualmetrics.py

```diff
@@ -1457,6 +1462,41 @@ def calculate_speed_index(progress):
         last_progress = p['progress'] / 100.0
     return int(si)
 
+def calculate_contentful_speed_index(progress, directory):
+    x = len(progress)
+    dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), directory)
+    first_paint_frame = os.path.join(
+        dir, "ms_{0:06d}.png".format(
+            progress[1]["time"]))
+    target_frame = os.path.join(
+        dir, "ms_{0:06d}.png".format(progress[x - 1]["time"]))
+    logging.debug("Target image for contSI is %s" % target_frame)
```

Comment:
Consider not abbreviating and putting the "tag" first, so: "contentfulSpeedIndex: target image is %s".